.github/workflows/buildi.yml

```
@@ -0,0 +1,7 @@
+- name: Android Build
```

actionlint (1.7.4)

workflow is sequence node but mapping node is expected

(syntax-check)